src/aspire/denoising/class_avg.py

```diff
@@ -290,12 +289,16 @@ def _images(self, indices):
 
         # Check if this src cached images.
         if self._cached_im is not None:
-            logger.debug(f"Loading {len(indices)} images from image cache")
-            im = self._cached_im[indices, :, :]
+            logger.debug(
+                f"Loading {len(indices)} images from image cache, indices {_indices}"
+            )
+            im = self._cached_im[_indices, :, :]
 
         # Check for heap cached image sets from class_selector.
         elif heap_inds:
-            logger.debug(f"Mapping {len(heap_inds)} images from heap cache.")
+            logger.debug(
+                f"Mapping {len(heap_inds)} images from heap cache, indices {indices}"
```

Comment:
Not `_indices`? I'm confused.

Reply:
I re-confirmed the `_indices` and `indices` in the code here.